ex04.ex

```diff
@@ -75,8 +75,9 @@ defmodule Ex04 do
   return value will be the thing you have to manipulate.
   """
 
-  def even_odd . . . "your code"
-
+  def even_odd_helper(true, num, {even, odd}), do: {even++[num], odd}
```

Comment:
This works, but for large lists it is really inefficient. It is better to prepend the number to the lists using `[num | odd]`, and then at the very end reverse the lists.